A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/hangar.flt

```diff
@@ -0,0 +1,312 @@
+A32NX_HYD_ENG1PUMP_FAULT=0
+A32NX_HYD_ENG1PUMP_TOGGLE=1
+A32NX_HYD_ENG2PUMP_FAULT=0
+A32NX_HYD_ENG2PUMP_TOGGLE=1
+A32NX_HYD_RATMANONLOCK_TOGGLE=0
+A32NX_HYD_ELECPUMP_FAULT=0
+A32NX_HYD_ELECPUMP_TOGGLE=1
+A32NX_HYD_ELECPUMPLOCK_TOGGLE=0
+A32NX_HYD_PTU_FAULT=0
+A32NX_HYD_PTU_TOGGLE=1
+A32NX_HYD_ELECPUMPY_FAULT=0
+A32NX_HYD_ELECPUMPY_TOGGLE=1
+A32NX_ENGMANSTART1LOCK_TOGGLE=0
+A32NX_ENGMANSTART2LOCK_TOGGLE=0
+A32NX_ENGMANSTART1_TOGGLE=0
+A32NX_ENGMANSTART2_TOGGLE=0
+A32NX_VENTILATION_BLOWER_FAULT=0
+A32NX_VENTILATION_BLOWER_TOGGLE=1
+A32NX_VENTILATION_EXTRACT_FAULT=0
+A32NX_VENTILATION_EXTRACT_TOGGLE=1
+A32NX_VENTILATION_CABFANS_TOGGLE=1
+A32NX_EMERELECPWR_EMERTESTLOCK_TOGGLE=0
+A32NX_EMERELECPWR_GEN1_FAULT=0
+A32NX_EMERELECPWR_GEN1_TOGGLE=1
+A32NX_EMERELECPWR_RAT_FAULT=0
+A32NX_EMERELECPWR_MANONLOCK_TOGGLE=0
+A32NX_EVAC_COMMANDLOCK_TOGGLE=0
+A32NX_EVAC_COMMAND_FAULT=0
+A32NX_EVAC_COMMAND_TOGGLE=1
+A32NX_EVAC_CAPT_TOGGLE=1
+A32NX_CARGOSMOKE_DISCH1LOCK_TOGGLE=0
+A32NX_CARGOSMOKE_DISCH2LOCK_TOGGLE=0
+A32NX_KNOB_SWITCHING_1_Position=1
+A32NX_KNOB_SWITCHING_2_Position=1
+A32NX_KNOB_SWITCHING_3_Position=1
+A32NX_KNOB_SWITCHING_4_Position=1
+A32NX_PANEL_DCDU_L_BRIGHTNESS=0.5
+A32NX_PANEL_DCDU_R_BRIGHTNESS=0.5
+A32NX_OVHD_HYD_BLUEPUMP_OVRD=0
+A32NX_OVHD_HYD_LEAK_MEASUREMENT_G=1
+A32NX_OVHD_HYD_LEAK_MEASUREMENT_B=1
+A32NX_OVHD_HYD_LEAK_MEASUREMENT_Y=1
+
+[Gauges.0]
+KollsmanSetting=29.921342849731445313
+
+[BleedAir.0]
+BleedAirSource=0
+EngineAirBleed.1=1
+EngineAirBleed.2=1
+APUAirBleed=0
+EngineRecievesAPUAirBleed.1=0
+EngineRecievesAPUAirBleed.2=0
+
+[Switches.0]
+PitotHeat=False
+PitotHeat_EX1=2
+WindshieldDeice=2
+BeaconLights=False
+LandingLights=False
+LogoLights=False
+NavLights=False
+PanelLights=True
+RecognitionLights=False
+StrobeLights=False
+TaxiLights=False
+WingLights=False
+CabinLights=False
+GlareshieldLights=False
+PedestalLights=False
+Potentiometer.7=0
+Potentiometer.84=0
+Potentiometer.85=0
+Potentiometer.86=0
+Potentiometer.87=0.1
+Potentiometer.88=0.1
+Potentiometer.89=0.1
+Potentiometer.90=0.1
+Potentiometer.91=0.1
+Potentiometer.92=0.1
+Potentiometer.93=0.1
+Potentiometer.96=0
+Potentiometer.97=0
+
+[AutoPilot.0]
+MasterSwitch=False
+WingLeveler=False
+Nav1Lock=False
+HeadingLock=False
+HeadingValue=0
+AltitudeLock=False
+AltitudeValue=5000
+AttitudeHold=False
+AirspeedHold=False
+AirspeedValue=100
+MachHold=False
+MachValue=0
+VerticalSpeedHold=False
+VerticalSpeedValue=0
+RPMHold=False
+RPMValue=0
+GlideslopeHold=False
+ApproachHold=False
+BackCourseHold=False
+YawDamper=False
+ToGa=False
+AutoThrottleArm=False
+GPSdrivesNAV1=False
+IsUsedForLesson=False
+ForceDisplayUI=False
+AltitudeSlotIndex=2
+SpeedSlotIndex=2
+VSSlotIndex=2
+
+[Controls.0]
+SpoilersHandle=000.00
+SpoilersArmed=False
+FlapsHandle=000.00
+LeftFlap=000.00
+RightFlap=000.00
+GearsHandle=000.01
+Gear1=100.00
+Gear2=100.00
+Gear3=100.00
+YokeY=050.00
+YokeX=050.00
+Rudder=050.00
+LeftBrake=100.00
+RightBrake=100.00
+ParkingBrake=0
```

Comment:
I would suggest parking brake 1 since there are no chokes in the hangar or anything attached to the wheels to keep the airplane stable.

ParkingBrake=1